### PR TITLE
Add infraJS docs

### PR DESCRIPTION
**Pull request by odacremolbap**
Add InfraJS documentation

### CODE REVIEW

File: docs/targets/infrajs.md [hunk 1]
```diff
@@ -0,0 +1,156 @@
+# InfraJS Target
+
+This event target does programatic transformations on received [CloudEvents][ce] using a Javascript function.
+
+## Prerequisites
+
+Basic knoledge of Javascript ES5 is required. Please note there are limitations to the interpreter:
```
Comment (sameersbn) [inline docs/targets/infrajs.md:7]:
```suggestion
Basic knowledge of JavaScript ES5 is required. Please note there are limitations to the interpreter:
```

File: docs/targets/infrajs.md [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+- `Timeout`  is the amount of milliseconds to wait before timing out the script.
+- `Type Loop Protection` when enabled will raise an error if the returned CloudEvent type is the same as the incoming one.
+
+### JS Script
+
+The JS Script field must include a function named `handle` with a single parameter for the incoming CloudEvent, and return the outgoing CloudEvent.
+
+- The incoming parameter is the [JSON][ce-json] representation of the incoming event.
+- The incoming parameter can be re-used as the outgoing parameter. That is useful if we want to modify a small set of the values in the incoming event.
+- The outgoing event must be a JSON structure that contains [context attributes][ce-context-attributes] at the root and [data payload][ce-data] at the `data` element. See [JSON format reference][ce-json].
```
Comment (sameersbn) [inline docs/targets/infrajs.md:29]: `ce-context-attributes` is  not defined and so is `ce-data` and `ce-json`

**Review by sameersbn — CHANGES_REQUESTED** [review]

File: docs/targets/infrajs.md (hunk 2 above)
Comment (odacremolbap) [reply docs/targets/infrajs.md:29]: thanks
https://github.com/triggermesh/docs/pull/80/commits/26edd9c10bc044187f7e8a0b45d234c7e0a88b59

**Review by sameersbn — APPROVED** [review]

**Comment by sameersbn** [comment]
lgtm

**Pull request merged**